I created a new virtualenv using python3 and ran `make requirements`, but it errors:
```
pip-compile -o requirements-python3/requirements.txt requirements/requirements.in
make: pip-compile: No such file or directory
make: *** [requirements] Error 1
```
Any idea what's wrong?

--- a/requirements-python3/dev-requirements.txt
+++ b/requirements-python3/dev-requirements.txt
@@ -78,6 +78,7 @@ eulxml==1.1.3
 fakecouch==0.0.15
 faker==0.8.15
 fixture==1.5.11
+flaky==3.6.0
 freezegun==0.3.11
 future==0.17.1            # via commonmark
 gevent==1.4.0

--- a/requirements-python3/test-requirements.txt
+++ b/requirements-python3/test-requirements.txt
@@ -72,6 +72,7 @@ ethiopian-date-converter==0.1.5
 eulxml==1.1.3
 fakecouch==0.0.15
 faker==0.8.15
+flaky==3.6.0
 freezegun==0.3.11
 gevent==1.4.0
 ghdiff==0.4

